cylc/flow/task_proxy.py

```diff
@@ -503,16 +503,16 @@ def state_reset(
             return True
         return False
 
-    def satisfy_me(self, prereqs) -> bool:
+    def satisfy_me(self, tokens: Tokens) -> bool:
         """Attempt to satisfy the given prerequisites.
 
         return True if all are valid, else False.
         """
-        bad = self.state.satisfy_me(prereqs)
+        bad = self.state.satisfy_me(tokens)  # TODO
```

Comment:
Ach, this was a partial suggestion, the TODO will addressing (it was passing a tuple, it's now passing tokens)

Reply:
Haha, actually no it was already taking tokens. The new function in task_pool converts from string to tokens, not the other way around. I just need to remove the TODO comment.

Reply:
(Done now)